PURL-SPECIFICATION.rst

```diff
@@ -364,8 +364,9 @@ To parse a ``purl`` string in its components:
   - Percent-decode each segment
   - Discard any '.' or '..' segment from that split
   - UTF-8-decode each segment if needed in your programming language
-  - Join segments back with a '/'
-  - This is the ``subpath``
+  - This list of path segments is the ``subpath``
+  - You may escape these path segments if needed by your environment (operating system, file system, programming language, shell, etc)
+  - You may join these path segments with the path delimiter of your environment (operating system, file system, etc)
```

Comment:
I am not sure these are necessary: implementations can store the list of strings anyway they like it.

Reply:
and since it might not be necessary, the **MAY** join ... this is a remark on implementations.

Reply:
IMHO it is not up to the specification to provide suggestions to implementors.

What I **would** expect from the specification is to provide a guideline on which language-specific methods should an implementation provide, e.g.:

- An implementation should allow users to extract the `subpath` as encoded ASCII string.

or:

- An implementation should allow users to extract the `subpath` as language-specific list of Unicode strings.